core/platform/email/gae_email_services_test.py

```diff
@@ -36,10 +45,170 @@ def test_sending_email(self):
             to=feconf.ADMIN_EMAIL_ADDRESS)
         self.assertEqual(0, len(messages))
 
+        messages = self.mail_stub.get_sent_messages(
+            to=feconf.ADMIN_EMAIL_ADDRESS)
+        self.assertEqual(0, len(messages))
         with self.swap(feconf, 'CAN_SEND_EMAILS', True):
             gae_email_services.send_mail(
-                feconf.SYSTEM_EMAIL_ADDRESS, feconf.ADMIN_EMAIL_ADDRESS,
+                feconf.SYSTEM_EMAIL_ADDRESS,
+                feconf.ADMIN_EMAIL_ADDRESS,
```

Comment:
Do we need this change?

Reply:
I did that in order to maintain consistency in the script.
In other places, I had to split them to multiple lines so as to not violate the Lint Rules.
Will change it back if you say so.
Thanks!

Reply:
I have reverted the change.
Thanks!